package.json

```diff
@@ -10,26 +10,25 @@
     "pretest": "npm run compile",
     "lint": "eslint src/*.js test-src/*.js",
     "lintfix": "eslint --fix src/*.js test-src/*.js",
-    "test": "npm run lint && npm run compile && NODE_ENV=test mocha test/*_test.js"
+    "test": "npm run lint && NODE_ENV=test mocha test/*_test.js",
+    "prepublish": "npm run compile"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/elenasolomon/azure-blob-storage.git"
   },
-  "dependencies": {
-    "assume": "^1.2.5",
-    "azure-storage": "^1.0.1",
+  "devDependencies": {
+    "babel-eslint": "^7.0.0",
     "babel-compile": "^2.0.0",
     "babel-preset-taskcluster": "^3.0.0",
-    "babel-runtime": "^6.6.1",
-    "debug": "^2.2.0",
     "eslint": "^2.10.2",
     "eslint-config-taskcluster": "^2.0.0",
-    "eslint-plugin-taskcluster": "^1.0.0",
+    "eslint-plugin-taskcluster": "^1.0.0"
+  },
+  "dependencies": {
+    "assume": "^1.2.5",
```

Comment:
I think we'll want to move this to `devDependencies` because in this case we only use it in developing the library.